spec/abstractops.html

```diff
@@ -65,6 +96,33 @@ <h1>ToTemporalDisambiguation ( _options_ )</h1>
     </emu-alg>
   </emu-clause>
 
+  <emu-clause id="sec-temporal-totemporalroundingmode" aoid="ToTemporalRoundingMode">
+    <h1>ToTemporalRoundingMode ( _options_ )</h1>
+    <emu-alg>
+      1. Set _options_ to ? NormalizeOptionsObject(_options_).
+      1. Return ? GetOption(_options_, *"roundingMode"*, *"string"*, « *"ceil"*, *"floor"*, *"trunc"*, *"nearest"* », *"nearest"*).
+    </emu-alg>
+  </emu-clause>
+
+  <emu-clause id="sec-temporal-totemporalroundingincrement" aoid="ToTemporalRoundingIncrement">
+    <h1>ToTemporalRoundingIncrement ( _options_, _dividend_, _inclusive_ )</h1>
+    <emu-alg>
+      1. Set _options_ to ? NormalizeOptionsObject(_options_).
+      1. If _dividend_ is *undefined*, then
```

Comment:
Why all this logic to handle _dividend_?  I think you can assert that it is a positive integer.

Reply:
There has to be some value that indicates that the increment doesn't have to divide evenly into anything (as in `dt1.difference(dt2, { smallestUnit: 'days', roundingIncrement: 10 })`.) It's not needed in this Part 1 pull request but it will be needed for Part 2 which adds rounding to the `difference()` methods.